codekata/compose.instructionset.md

```diff
@@ -0,0 +1,354 @@
+## Login UI Test with Compose
+
+This section is equivalent to the one with Login UI Test from robolectric.instructionset.md. 
+Make sure to read that one first as this one only focuses on the differences that Compose brings.
+We will write the same tests so that we see clearly the differences between the two.
+
```

Comment:
I think it would be a great place to specify which TestClass we are talking about next.

Reply:
You're right. Added.